v4/weaviate/backup/backup_restorer.go

```diff
@@ -0,0 +1,46 @@
+package backup
+
+import (
+	"context"
+
+	"github.com/semi-technologies/weaviate/entities/models"
+)
+
+type BackupRestorer struct {
+	helper            *backupRestoreHelper
+	className         string
+	storageName       string
+	backupID          string
+	waitForCompletion bool
+}
+
+// WithClassName specifies the class which should be restored
+func (r *BackupRestorer) WithClassName(className string) *BackupRestorer {
+	r.className = className
+	return r
+}
```

Comment:
Restore backup also requires include / exclude classes similarly to Create backup
See https://github.com/semi-technologies/weaviate/issues/2147#issuecomment-1230229508